cmd/terradoc/cli/validate.go

```diff
@@ -80,17 +108,40 @@ func (vcm ValidateCmd) Run() error {
 	return nil
 }
 
-func printValidationSummary(summary validators.Summary, docFilename, defFilename string) {
+func printValidationSummary(summary validators.Summary, docFilename string) {
 	for _, missingDef := range summary.MissingDefinition {
-		fmt.Fprintf(os.Stderr, "Missing %s definition: %q is not defined in %q\n", summary.Type, missingDef, defFilename)
+		fmt.Fprintf(os.Stderr, "Missing %s definition: %q is not defined in .tf files\n", summary.Type, missingDef)
```

Comment:
this is the other way around: it is NOT a missing definition but a documentation for a non existing variable.

```suggestion
		fmt.Fprintf(os.Stderr, "Unknown %s documented: %q is not defined in any .tf files\n", summary.Type, missingDef)
```